src/elf.rs

```diff
@@ -411,11 +421,22 @@ impl<'a> Elf<'a> {
         self.symbols.insert(idx, symbol);
         self.section_symbols.insert(idx, section_symbol);
         // FIXME: probably add padding alignment
+
         let mut section = {
+            let stype =
+                if prop.function {
+                    SectionType::Bits
+                } else if prop.cstring {
+                    SectionType::String
+                } else {
```

Comment:
I’d prefer explicit match here in the prop field in case this gets changed but it’s fine for now